### PR TITLE
*: Add --log-file flag to set log file path

### PR DESCRIPTION
**Pull request by shenli**
@coocood @siddontang @qiuyesuifeng PTAL
#1559

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by c4pt0r** [comment]
LGTM

File: tidb-server/main.go [hunk 1]
```diff
@@ -75,7 +75,18 @@ func main() {
 		ReportStatus: *reportStatus,
 	}
 
+	// Set log options
```
Comment (zimulala) [inline tidb-server/main.go]: add . or Set -> set

Comment (shenli) [inline tidb-server/main.go]: PTAL @zimulala

**Comment by zimulala** [comment]
LGTM

**Pull request merged**